public/app/plugins/datasource/prometheus/datasource.ts

```diff
@@ -103,6 +103,10 @@ export class PrometheusDatasource {
   interpolateQueryExpr(value, variable, defaultFormatFn) {
     // if no multi or include all do not regexEscape
     if (!variable.multi && !variable.includeAll) {
+      if (typeof value === 'number') {
+        // $__interval_ms type is number, convert to string here
+        value = value.toString();
```

Comment:
not sure this is the correct place to solve this, maybe the root issue should be fixed instead? wonder if allowing $__interval_ms to have number value is not correct but it should be turned into string when it is save in scopedVars (in metrics_ctrl & prometheus datasource.ts)